.github/workflows/dev_on_push_workflow_main.yml

```diff
@@ -114,7 +114,7 @@ jobs:
       GCP_EHP_WORKLOAD_IDENTITY_PROVIDER: ${{ secrets.GCP_EHP_WORKLOAD_IDENTITY_PROVIDER }}
   hard-deploy-android-testing:
     needs: yarn-tester
-    if: startsWith(github.ref, 'refs/tags/testing') || startsWith(github.ref, 'refs/tags/v')
+    if: startsWith(github.ref, 'refs/tags/testing')
```

Comment:
avec le nouveau workflow, on va créer des tags

mais les tags ne seront plus déployés

on le supprime pour quelles raisons ?

Reply:
Comment ça les tags ne seront plus déployés ?

Reply:
Comme mentionné dans le message de commit cette condition lançait 2 fois le déploiement hard de testing, un pour l'ancienne version ité n (déploiement inutile) et un pour la version ité n+1

Reply:
si t'es certaine que c'est inutile, ça me va 👌